docs/sources/installation/mac.md

```diff
@@ -9,49 +9,50 @@ weight = 500
 +++
 
 
-# Installing on macOS
+# Install on macOS
 
-This page provides instructions to help you install Grafana on macOS. You can either install using homebrew or a binary tar file.
+This page provides instructions to help you install Grafana on macOS. You can either install using Homebrew or a binary tar file.
 
 
-## Install using homebrew
+## Install with Homebrew
 
-Installation can be done using [homebrew](http://brew.sh/)
+Installation can be done using [homebrew](http://brew.sh/).
 
-Install latest stable:
+Install latest stable version:
 
 ```bash
 brew update
 brew install grafana
 ```
 
-To start grafana look at the command printed after the homebrew install completes.
-
-To upgrade use the reinstall command
+You can also install the latest unstable Grafana version from Git:
 
 ```bash
-brew update
-brew reinstall grafana
+brew install --HEAD grafana/grafana/grafana
 ```
 
--------------
+To start Grafana, run the command printed after the Homebrew install completes.
 
-You can also install the latest unstable grafana from git:
+## Upgrade with Homebrew
 
+To upgrade Grafana, use the reinstall command:
 
 ```bash
-brew install --HEAD grafana/grafana/grafana
+brew update
+brew reinstall grafana
 ```
 
-To upgrade grafana if you've installed from HEAD:
+To upgrade grafana if you installed an unstable version from Git:
 
 ```bash
 brew reinstall --HEAD grafana/grafana/grafana
 ```
 
 ### Start Grafana
 
-To start Grafana using homebrew services first make sure homebrew/services is installed.
+To start Grafana using Homebrew services: 
```

Comment:
Is Homebrew Services a separate installation from Homebrew? Would the user ever have Homebrew installed (and thus be able to perform the earlier installation) without having the services installed?

Reply:
@marcusolsson or @torkelo ?

Reply:
Homebrew services is installed on the first use, i.e. upon running `brew services`.